department-of-veterans-affairs/caseflow
# Craig/appeals 15567

Resolves [APPEALS-15567](https://vajira.max.gov/browse/APPEALS-15567)

### Description
- Added _cavc_dashboard.scss file to customize the TabWindow styles without affecting the other TabWindow instances

### Acceptance Criteria
- [ ] All tabs have complete borders (gray unselected, blue selected)
- [ ] Bottom row of tabs always has equal or more tabs than the other rows to remove whitespace below tabs/tab overhang
- [ ] Tabs are aligned consistently

### Testing Plan
1. Go to the CAVC Dashboard of an appeal with multiple dashboard tabs
2. Resize the window from large to small to ensure that the tabs behave as described above
